src/test/store/actions.test.js

```diff
@@ -460,7 +460,7 @@ describe('selectors/getCombinedTimingRows', function () {
         start: [0],
         end: [10],
         index: [0],
-        label: ['renderFunction'],
+        label: [expect.any(Function)],
```

Comment:
I tried to use `makeMarkerTimingSerializable` on this list, but Flow was really getting in the way. Our type `CombinedTimingRows` isn't great because it doesn't have a string property to distinguish the 2 possible types in the union. 

So after spending 1h on this I decided to go the easy way instead.